### DISCUSSION TITLE
Welcome to Yugioh-CardDex Discussions!

### DISCUSSION THREAD

**Discussion by Gianguyen1234** [Gianguyen1234/Yugioh-CardDex#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by thaonguyen23546** [comment]
Love it

**Comment by VuongKha** [comment]
I'm waiting to try your YGO Card Dex
